packages/terra-docs/src/Notice.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './Notice.module.scss';
+
+const cx = classNames.bind(styles);
+
+const getTitle = (variant) => {
+  if (variant === 'deprecation') {
+    return 'Deprecation Notice';
+  }
+
+  return 'error';
+};
+
+const propTypes = {
+  /**
+  * The main body text of the notice.
+  */
+  children: PropTypes.string,
+  /**
+  * The type of notice.
+  */
+  variant: PropTypes.oneOf(['deprecation']),
+};
+
+const defaultProps = {
+  variant: 'deprecation',
+};
+
+const Notice = ({
+  variant,
+  children,
+}) => {
+  const theme = React.useContext(ThemeContext);
+  return (
+    <h2 className={cx('notice', variant, theme.className)}>
```

Comment:
I set this as an h2.. Do I need to open up the level of the heading to be a prop? I anticipate the notice generally being used directly after the initial h1 on a page.

Reply:
I don't think it's worth opening this up quite yet, but it wouldn't hurt anything.